content/notices/2022-07-20-atlassian.md

```diff
@@ -0,0 +1,14 @@
+---
+Title: Servlet Filter Bypass Vulnerabilities in Multiple Atlassian Products (CVE-2022-26136, CVE-2022-26137)
+Date: 2022-07-20 10:00
+Category: Services & System Applications
+Tags: Confluence, Jira, Bitbucket, Bamboo, Fisheye, Crowd, Atlassian, Very High Severity
+Slug: atlassian
+Summary: Servlet Filter Bypass Vulnerabilities have been found in multiple Atlassian products that may enable Authentication Bypasses, XSS Vulnerabilities, and CORS vulnerabilities. These vulnerabilites have been assigned a bug alert severity of 'very high'. Atlassian recommends patching affected installations immediately.
+---
+
+On Wednesday, July 20th, 2022, [Atlassian released a Security Advisory](https://confluence.atlassian.com/doc/confluence-security-advisory-2022-06-02-1130377146.html) stating that Servlet Filter bypasses exist in both first and third party applications tied to multiple Atlassian products. At this time they have not released any specifics as to what the exact vulnerable endpoint is, or any indicators of compromise that could lead defenders to believe they have been exploited. The current fix is to patch to the level indicated in the advisory.
```

Comment:
> On Wednesday, July 20th, 2022, [Atlassian released a Security Advisory](https://confluence.atlassian.com/doc/confluence-security-advisory-2022-06-02-1130377146.html)

Is this the right link? The latest update in this one seems to have been on June 10th.